common/path.c

```diff
@@ -122,12 +123,12 @@ expand_homedir (const char *remainder)
 	if (remainder != NULL &&
 	    strncmp (remainder, ".config", 7) == 0 &&
 	    is_path_component_or_null (remainder[7])) {
-		env = getenv ("XDG_CONFIG_HOME");
+		env = secure_getenv ("XDG_CONFIG_HOME");
```

Comment:
I vaguely remember that we deliberately use `getenv` instead of `secure_getenv` here, because this code is only used by p11-kit-client.so, and it can be loaded into a setuid'ed binary.